docs/references/configuration.rst

```diff
@@ -774,6 +774,24 @@ raw-media-types
 
    raw-media-types="image/png, font/woff2"
 
+.. _server_cors_allowed_origins:
+
+server-cors-allowed-origins
+---------------------------
+
+  =============== ===================================
+  **Type**        String
+  **Default**     `n/a` 
+  **Reloadable**  N
+  **Environment** PGRST_SERVER_CORS_ALLOWED_ORIGINS
+  **In-Database** `pgrst.server_cors_allowed_origins`
+  =============== ===================================
+
+  Specifies allowed CORS origins in this config. See :ref:`cors`.
+
+  * When this is not set, PostgREST **accepts** CORS requests from any domain
+  * When this is set to :code:`""`, PostgREST **rejects** CORS requests from any domain
```

Comment:
@taimoorzaeem I wonder if this is useful behavior. Wouldn't be better to treat `""` the same as not set - accept all reqs from any domain?

Reply:
Ah, that was my suggestion in https://github.com/PostgREST/postgrest/pull/2986#pullrequestreview-1694030855. It's not so much that rejects any request, i.e. the ones with empty "Origin: " will work (kinda impossible case). In other words, when this is set then the values will be used as-is. The note could be: "To allow all origins do not set this config, otherwise, all the values inside will be used as-is".

On the other hand, I get that it may not be too intuitive, maybe we could add this special case for "" or even "*" (which will be invalid if the config has more values for any reason "*,http://google.com,...").

Reply:
@laurenceisla Got it. As you mentioned is like an empty search path. But it does look "sloppy" and perhaps some user could interpolate an empty value `..="$EMPTY"` in the confg and be left surprised as to why all requests are failing?

I believe we should default `""` to accept all. WDYT?

Reply:
A quick look on the `db-extra-search-path` returned `[""]` when empty. But it does not give any issue if that's the case.

> I believe we should default "" to accept all. WDYT?

Yes, agree. Treat only an empty string as if it wasn't set at all. Any other value, would be considered as-is.

Reply:
> IIRC we did the same for many other configs "" translates to Nothing?

Funny that I tried setting `db-schemas = ""` to test this functionally,  and it worked OK when requesting tables, because it defaulted to the `public` schema (as if it wasn't set at all). Then, while working on another issue with RPCs, it gave me this error:

`message: "Could not find the function .index without parameters in the schema cache"`

Had me scratching my head for a while, then remembered the empty setting. So it looks like it's not defaulting to `public` in the RPC search... odd.

Reply:
I implemented this in #3028. Now the last line can be removed, and the info on the empty strings should be added to the first one.